### ISSUE TITLE
renal tubular  acidosis

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/mondo#1825]
The renal tubular  acidosis section needs attention. Probably it would be good to have children reflecting these subtypes:
https://www.niddk.nih.gov/health-information/kidney-disease/renal-tubular-acidosis

**Comment by nicolevasilevsky** [comment]
The subtypes listed are:
- Type 1: Classical Distal RTA (MONDO_0011268) -> _I added Classical Distal RTA and Type 1 RTA as exact synonyms_
- Type 2: Proximal RTA (MONDO_0008369) -> _I added Type 2 RTA as an exact synonym_
- Type 4: Hyperkalemic RTA -> _I added this as a new term (MONDO_0100161)_

**Comment by nicolevasilevsky** [comment]
@pnrobinson is type 1 classic distal RTA equiv to https://www.omim.org/entry/602722?

**Comment by pnrobinson** [comment]
@nicolevasilevsky  There are hereditary and acquired forms of type 1 distal RTA. The OMIM entry should be a child.
This is another instance where one can regard type 1 distal RTA as a phenotypic feature of a disease or as a disease, depending on context. @cmungall @mellybelly  We need to figure out how to deal with situations like this better. As a first step, I would suggest that we make some relationship between HPO terms such as https://hpo.jax.org/app/browse/term/HP:0008341 and corresponding MONDO terms. This could be another argument for the utility of MONDO.

**Comment by nicolevasilevsky** [comment]
@pnrobinson and @cmungall 

Should MONDO_0018440 'autosomal recessive distal renal tubular acidosis' and MONDO_0011268 'renal tubular acidosis, distal, autosomal recessive' be merged?

I think these are the same.

Related to #1764

**Comment by pnrobinson** [comment]
dominant
--- 17q21.31 | Distal renal tubular acidosis 1 | AD | 3 | 179800 | SLC4A1 | 109270
recessive
---  2p13.3 | Distal renal tubular acidosis 2 with progressive sensorineural hearing loss | AR | 3 | 267300 | ATP6V1B1 | 192132
---  7q34 | Distal renal tubular acidosis 3, with or without sensorineural hearing loss | AR | 3 | 602722 | ATP6V0A4 | 605239
---   17q21.31 | Distal renal tubular acidosis 4 with hemolytic anemia (equivalent to existing class 
)

**Comment by nicolevasilevsky** [comment]
the terms above should not be merged (because then a OMIM would be a parent of an OMIM).
Renamed to OMIM name.
